pkgs/development/python-modules/hypothesis/default.nix

```diff
@@ -4,59 +4,66 @@
 , fetchFromGitHub
 , attrs
 , pexpect
-, doCheck ? true
+, doCheck ? false
 , pytestCheckHook
 , pytest-xdist
 , sortedcontainers
 , pythonOlder
 }:
 
-buildPythonPackage rec {
-  pname = "hypothesis";
-  version = "6.46.10";
-  format = "setuptools";
+let
+  self = buildPythonPackage rec {
+    pname = "hypothesis";
+    version = "6.46.10";
+    format = "setuptools";
 
-  disabled = pythonOlder "3.7";
+    disabled = pythonOlder "3.7";
 
-  src = fetchFromGitHub {
-    owner = "HypothesisWorks";
-    repo = "hypothesis";
-    rev = "hypothesis-python-${version}";
-    hash = "sha256-eQ7Ns0k1hOVw8/xiINMei6GbQqDHXrBl+1v8YQeFO9Q=";
-  };
+    src = fetchFromGitHub {
+      owner = "HypothesisWorks";
+      repo = "hypothesis";
+      rev = "hypothesis-python-${version}";
+      hash = "sha256-eQ7Ns0k1hOVw8/xiINMei6GbQqDHXrBl+1v8YQeFO9Q=";
+    };
+
+    sourceRoot = "${src.name}/hypothesis-python";
 
-  postUnpack = "sourceRoot=$sourceRoot/hypothesis-python";
+    propagatedBuildInputs = [
+      attrs
+      sortedcontainers
+    ];
 
-  propagatedBuildInputs = [
-    attrs
-    sortedcontainers
-  ];
+    checkInputs = [
+      pexpect
+      pytest-xdist
+      pytestCheckHook
+    ];
 
-  checkInputs = [
-    pexpect
-    pytest-xdist
-    pytestCheckHook
-  ];
+    inherit doCheck;
 
-  inherit doCheck;
+    # This file changes how pytest runs and breaks it
+    preCheck = ''
+      rm tox.ini
+    '';
```

Comment:
Normally we only remove the problematic arguments because sometimes some of them are actually required